### PR TITLE
[FIX] l10n_ar: remove inconsistent tax demo data

### PR DESCRIPTION
**Pull request by clbr-odoo**
Remove tax demo data:
- They have the same tax name which breaks a uniqueness constraint.
- It doesn't make sense to have taxes demo data

### CODE REVIEW

File: addons/l10n_ar/demo/account_tax_demo.xml [hunk 1]
```diff
@@ -1,102 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smetl) [inline addons/l10n_ar/demo/account_tax_demo.xml:1]: Remove it from the manifest as well ;)

Comment (clbr-odoo) [reply addons/l10n_ar/demo/account_tax_demo.xml:1]: @smetl My bad, ready :)

**Comment by clbr-odoo** [comment]
@smetl I removed places where those taxes were assigned and module seems to install correctly now.

**Comment by smetl** [comment]
@robodoo r+

**Pull request closed without merging**